org.idempiere.test/src/org/idempiere/test/form/CreateFromDepositBatchFormTest.java

```diff
@@ -165,7 +165,7 @@ public void loadPayments(Integer BankAccount, Integer BPartner, String DocumentN
 				Timestamp DateFrom, Timestamp DateTo, BigDecimal AmtFrom, BigDecimal AmtTo, Integer DocType,
 				String TenderType, String AuthCode) {
 			Vector<Vector<Object>> datas = super.getBankAccountData(BankAccount, BPartner, DocumentNo, DateFrom, DateTo, AmtFrom, AmtTo, DocType,
-					TenderType, AuthCode);
+					TenderType, AuthCode, 0);
```

Comment:
Even if I set C_Charge_ID, this unit test fails because `getBankAccountData` does not return any data.